exporter/splunkhecexporter/metricdata_to_splunk.go

```diff
@@ -213,7 +213,7 @@ func mapSummaryValue(metric *metricspb.Metric, summaryValue *metricspb.SummaryVa
 	if percentiles == nil {
 		return values, fmt.Errorf(
 			"unknown percentiles values for summary metric %q",
-			*metric)
+			metricName)
```

Comment:
@atoulme as the original author of this exporter, please confirm if this fix is correct.

Reply:
Looks good, it is just an error.